src/database/effects/cards/2-0-323.ts

```
@@ -5,63 +5,52 @@ import type { CardEffects, StackWithCard } from '../classes/types';
-      }
+    switch (choice) {
+      case '1':
+        const deck = stack.processing.owner.deck.filter(
+          card =>
+            card.catalog.cost <= 3 &&
+            card.catalog.species?.includes('魔導士') &&
+            card.catalog.type === 'unit'
+        ) as Unit[];
+        if (deck.length > 0) {
+          const [randomCard] = EffectHelper.random(deck, 1);
+          await System.show(
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)